src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -68,13 +68,17 @@ static void GetLoopInformation(uv_timer_s *data) {
 #else
 static void GetLoopInformation(uv_timer_s *data, int status) {
 #endif
+	double mean = std::numeric_limits<double>::infinity();
```

Comment:
I think this should be 0 rather than infinity.  I know technically dividing a number by 0 equals infinity but in terms of calculating mean values using infinity is not right.  It will also look really weird if we try to put this data in a graph.